website/content/prometheus-metrics/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Prometheus Metrics
+weight: 11
+---
+
+## MetalLB K8S client metrics
+
+| Name                                   | Description                                                                      |
+| -------------------------------------- | -------------------------------------------------------------------------------- |
+| metallb_k8s_client_updates_total       | Number of k8s object updates that have been processed                            |
+| metallb_k8s_client_update_errors_total | Number of k8s object updates that failed for some reason                         |
+| metallb_k8s_client_config_loaded_bool  | 1 if the MetalLB configuration was successfully loaded at least once             |
+| metallb_k8s_client_config_stale_bool   | 1 if running on a stale configuration, because the latest config failed to load  |
+
+## MetalLB BGP metrics
+#### Note: metrics labeled by BGP peer
```

Comment:
Can you expand this providing an example of labels to make it more clear?
I'd write something like:
"each metric contains labels that refert to the bgppeer" and provide an example

Reply:
![image](https://user-images.githubusercontent.com/40122521/201833356-764f31c6-ca7a-4a87-9ff6-8fb14ba8f0c0.png)

Reply:
@fedepaol What do you think about this explanation and example?

Reply:
done